### ISSUE TITLE
Update cowlib dependency

### ISSUE THREAD

**Issue by bruno-b-martins** [PSPDFKit-labs/bypass#110]
Hello there 👋 

I've just added bypass to my Phoenix project and I get the following error when I run `mix deps.get`:
```
Failed to use "cowlib" because
  bypass (version 2.0.0) requires ~> 1.0.1
  mix.lock specifies 2.9.1
```

I really don't want to downgrade my cowlib dependency (which actually might break some other dependencies...)
Would it be possible to upgrade the `cowlib` dependency on your side or does it break some other stuff somewhere? 😓

**Comment by feliperenan** [comment]
Hi 👋 

Just wanted to say that this https://github.com/PSPDFKit-labs/bypass/pull/101 fix this issue. I was facing the same issue and I was able to solve it just pointing Bypass to this PR branch.

**Comment by BobbyMcWho** [comment]
https://github.com/PSPDFKit-labs/bypass/pull/101#issuecomment-721610660
There is a RC available "2.1.0-rc.0"

**Comment by ream88** [comment]
`v2.1.0` is out!
